packages/manager/apps/dedicated/client/app/ip/components/list/popover/additional-ipv6.html

```diff
@@ -0,0 +1,60 @@
+<div data-uib-dropdown>
```

Comment:
bootstrap is deprecated. Take a look to `ovh-ui-kit` https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-dropdown-webcomponent--default

Reply:
ok